controls/map/layers/overview.md

```diff
@@ -1,26 +1,27 @@
 ---
 title: Layers Overview
 page_title: .NET MAUI Map Documentation - Map Layers Overview
-description: Check our &quot;Layers Overview&quot; documentation article for Telerik Map for .NET MAUI control.
+description: Learn more about the Map Layers used in Telerik UI for .NET MAUI Map control.
 position: 0
 slug: map-layers-overview
 ---
 
 # .NET MAUI Map Layers
 
-The Map control visualizes spatial data using *layered architecture*. It could display spatial data points (e.g. location of cities), polylines (e.g. road connections), or polygons (shape of countries or continents) in a *geographic coordinate system*. In this coordinate system, a point on the surface of the earth is defined using latitudes and longitudes.
+The Map control visualizes spatial data using *layered architecture*. It can display spatial data points
```

Comment:
- Don't use italics for emphasis
- Don't use a line break before ending the sentence.